app/models/jira_issue.rb

```diff
@@ -0,0 +1,42 @@
+# == Schema Information
+#
+# Table name: jira_issues
+#
+#  id              :bigint           not null, primary key
+#  environment     :enum
+#  informed_at     :datetime         not null
+#  issue_type      :enum             not null
+#  key             :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  jira_project_id :bigint           not null
+#
+# Indexes
+#
+#  index_jira_issues_on_jira_project_id  (jira_project_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (jira_project_id => jira_projects.id)
+#
+
+class JiraIssue < ApplicationRecord
```

Comment:
no need to change it, but maybe `JIRA` could have been a namespace?

Reply:
hey, @santib I found [this](https://stackoverflow.com/questions/40478054/rails-5-namespacing-models) answer in StackOverflow about namespacing models. What are your thoughts? Is it worth doing the refactor?